### PR TITLE
Add Lixee ZiPulses

### PR DESCRIPTION
**Pull request by ralmn**
Add Lixee ZiPulses 

https://github.com/fairecasoimeme/ZiPulses

PR Docs: https://github.com/Koenkk/zigbee2mqtt.io/pull/1958

### CODE REVIEW

File: devices/lixee.js [hunk 1]
```diff
@@ -688,4 +755,25 @@ const definition = {
...
+    ],
+    configure: async (device, coordinatorEndpoint, logger) => {
+        const endpoint = device.getEndpoint(1);
+        const binds = ['genPowerCfg', 'seMetering', 'msTemperatureMeasurement'];
+        await reporting.bind(endpoint, coordinatorEndpoint, binds);
+        await endpoint.read('seMetering', ['divisor', 'unitOfMeasure', 'multiplier']);
+    },
+};
+
+module.exports = [definitionZlinky, definitionZiPulses];
```
Comment (Koenkk) [inline devices/lixee.js:779]: Please inline the definitions (like e.g. https://github.com/Koenkk/zigbee-herdsman-converters/blob/2e2c21f08916fc17727a108adc2306d952c53288/devices/awox.js#L63)

**Comment by ralmn** [comment]
I just made the fixes

**Comment by Koenkk** [comment]
Thanks!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**